ruma-serde/src/canonical_json.rs

```diff
@@ -97,4 +114,47 @@ mod test {
             r#"{"auth":{"mxid":"@john.doe:example.com","profile":{"display_name":"John Doe","three_pids":[{"address":"john.doe@example.org","medium":"email"},{"address":"123456789","medium":"msisdn"}]},"success":true}}"#
         )
     }
+
+    #[test]
+    fn check_serialize_map_to_canonical() {
+        let mut expected = BTreeMap::new();
+        expected.insert("foo".into(), CanonicalJsonValue::String("string".into()));
+        expected.insert(
+            "bar".into(),
+            CanonicalJsonValue::Array(vec![
+                CanonicalJsonValue::Integer(js_int::int!(0)),
```

Comment:
Let's `use js_int::int;` in this module and shorten these to

```suggestion
                CanonicalJsonValue::Integer(int!(0)),
```

Reply:
You only updated the test below :smile: